src/software/ai/hl/stp/play/play.cpp

```diff
@@ -403,6 +419,59 @@ Play::assignTactics(const WorldPtr &world_ptr, TacticVector tactic_vector,
         current_tactic_robot_id_assignment};
 }
 
+std::vector<Play::RobotSubstituionParam> Play::trySubstituteInjuredRobots(const WorldPtr& world_ptr){
+    // Don't do anything if there aren't any injured robots  
+    const std::vector<Robot>& injured_robots = world_ptr->friendlyTeam().getInjuredRobots();
+    if(injured_robots.size()  == 0)
+        return {};
+
+    /* substitute injured robots when the game is in play */
+    std::vector<RobotSubstituionParam> substitution_robot_param = {};
+    std::map<RobotId, TrajectoryPath> robot_trajectories = {};
+    std::size_t num_injured = injured_robots.size();
+    std::size_t substitution_count = std::min(num_injured, 
+            static_cast<std::size_t>(ai_config.robot_substitution_config().maximum_robot_substituion_count()));
+
+    // substitution tactic is just a move tactic to a decided location
+    std::shared_ptr<MoveTactic> auto_sub_tactic = std::make_shared<MoveTactic>();
+
+    // move to middle of court and to positive y boundary and stop
+    double y_coordinate = world_ptr->field().totalYLength() / 2.0; 
+    if(!ai_config.robot_substitution_config().substitute_at_positive_y())
```

Comment:
using explicit `{ }` is a part of our code style guide: https://github.com/UBC-Thunderbots/Software/blob/bd3317f338c7086fdb9dd05c0db8a11e601efe26/docs/code-style-guide.md?plain=1#L307